.storybook/webpack.config.js

```diff
@@ -8,15 +8,24 @@ module.exports = {
         use:
           'react-native-web-image-loader?name=[name].[ext]&scalings[@2x]=2&scalings[-3x]=3',
       },
+      {
+        test: /\.js$/,
```

Comment:
Do you think we can remove this now, @lukewalczak ?
It should not be necessary, right?

Reply:
Right, now it's redundant.